.circleci/config.yml

```diff
@@ -8,20 +8,17 @@ executors:
 jobs:
   verify_python:
     docker:
-      - image: circleci/python:3.8.12
+      - image: cimg/python:3.10.5
```

Comment:
for anyone else wondering about this change:

https://circleci.com/developer/images/image/cimg/python

> This image is designed to supercede the legacy CircleCI Python image, circleci/python.